cmd/containerd/main.go

```diff
@@ -59,11 +55,6 @@ high performance container runtime
 			Value: "shim",
 		},
 		cli.StringFlag{
-			Name:  "socket, s",
-			Usage: "socket path for containerd's GRPC server",
-			Value: "/run/containerd/containerd.sock",
-		},
-		cli.StringFlag{
 			Name:  "metrics-address, m",
 			Usage: "tcp address to serve metrics on",
 			Value: "127.0.0.1:7897",
```

Comment:
Agreed. There are dozens of future changes still to do. This really is the first baby step. I really want to be incremental and do small easily reviewable changes, not address everything in a few large PRs - small PRs are much easier to scope and review. So yes, I am aware of this, just not now.